doc/source/whatsnew/v0.25.0.rst

```diff
@@ -391,6 +391,7 @@ Groupby/Resample/Rolling
 - Bug in :meth:`pandas.core.window.Rolling.min` and :meth:`pandas.core.window.Rolling.max` that caused a memory leak (:issue:`25893`)
 - Bug in :meth:`pandas.core.groupby.GroupBy.idxmax` and :meth:`pandas.core.groupby.GroupBy.idxmin` with datetime column would return incorrect dtype (:issue:`25444`, :issue:`15306`)
 - Bug in :meth:`pandas.core.groupby.GroupBy.cumsum`, :meth:`pandas.core.groupby.GroupBy.cumprod`, :meth:`pandas.core.groupby.GroupBy.cummin` and :meth:`pandas.core.groupby.GroupBy.cummax` with categorical column having absent categories, would return incorrect result or segfault (:issue:`16771`)
+- Bug in :meth:`pandas.core.groupby.SeriesGroupBy.transform` where transforming an empty SeriesGroupBy would raise error. Now returns empty Series, just like .apply and .agg. (:issue:`26208`)
```

Comment:
no need to say SeriesGroupBy, users's don't have much idea what this mean.  rather

Bug in :meth:`pandas.core.groupby.SeriesGroupBy.transform` where transforming an empty group would incorrectly raise

is enough

Reply:
Could we somehow use `Series.groupby` instead of `SeriesGroupBy` in this `whatsnew` note, since people won't be instantiating a `SeriesGroupBy` object directly?

Reply:
I've change the comment to be more aligned to jreback's feedback. Hope this also addresses gfyoung's concern. Think we should, however, maintain the reference "Bug in :meth:`pandas.core.groupby.SeriesGroupBy.transform`" because that's actually where the problem appears.